src/app/AppModel.ts

```diff
@@ -9,7 +9,7 @@
  * Message format for simulator app
  */
 export interface AppMessage {
-  appId: 'GPS' | 'FLIP' | 'TEMP';
+  appId: 'GPS' | 'FLIP' | 'TEMP' | 'DEVICE';
```

Comment:
```suggestion
  appId: 'GPS' | 'FLIP' | 'TEMP' | 'DEVICE_INFO';
```

`DEVICE` is too generic.

Reply:
Agreed. See comment.